integration/docker/build_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docker
+
+import (
+	. "github.com/clearcontainers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"os"
+	"path/filepath"
+)
+
+const dockerFile = "src/github.com/clearcontainers/tests/Dockerfiles/."
+
+var _ = Describe("build", func() {
+	var (
+		args []string
+		id   string
+		name string = "test"
+	)
+
+	BeforeEach(func() {
+		id = randomDockerName()
+	})
+
+	AfterEach(func() {
+		args = []string{"rmi", name}
+		runDockerCommand(0, args...)
+		Expect(ContainerExists(id)).NotTo(BeTrue())
+	})
+
+	Describe("build with docker", func() {
+		Context("docker build env vars", func() {
+			It("should display env vars", func() {
+				gopath := os.Getenv("GOPATH")
+				entirePath := filepath.Join(gopath, dockerFile)
+				args = []string{"build", "-t", name, entirePath}
```

Comment:
if the Dockerfile name is changed,  we should add `-f ` flag to specify the path of the dockerfile: E.g. `-f Dockerfile.buildtest`